nss-day-cohort-48/rare-server-river-rats
# fixed create category function to work with sql

#### Changes Made
1. Added file `request.py` to `categories` directory.
1. Modified file `categories/request.py` to include `create category` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ken-server
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > make sure the create category is working with sql and looks correct
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
